ci/lava/README.md

```diff
@@ -0,0 +1,17 @@
+# Test setup
+
+`ci/lava` directory contains directories with names corresponding to build MACHINE names.
+New directories should only be added after the build is available for a given MACHINE.
+All files in the `ci/lava/<MACHINE>` directory should have `.yaml` extension.
+Each file should be a valid LAVA job template.
+
```

Comment:
Maybe add a mention that all yaml files in there will be discovered dynamically and executed during the repo CI (PR, Push, nightly)?

Reply:
Added a paragraph to README.